gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/iceberg/IcebergCatalogFactory.java

```diff
@@ -18,14 +18,19 @@
 package org.apache.gobblin.data.management.copy.iceberg;
 
 import org.apache.hadoop.conf.Configuration;
-import org.apache.iceberg.hive.HiveCatalogs;
+import org.apache.iceberg.hive.HiveCatalog;
+
+import com.google.common.collect.Maps;
 
 
 /**
  * Provides an {@link IcebergCatalog}.
  */
 public class IcebergCatalogFactory {
   public static IcebergCatalog create(Configuration configuration) {
-    return new IcebergHiveCatalog(HiveCatalogs.loadCatalog(configuration));
+    HiveCatalog hcat = new HiveCatalog();
```

Comment:
actually `HiveCatalogs` is deprecated and got removed in iceberg 0.12 (we're on 0.11 currently).  I attempted upgrading beyond 0.11 and found that quite easy (ITO migrating source code).  even though we remain on 0.11, I left this change in, because it's stated to be an equivalent replacement, and it will be one less thing to do when we're ready for  >0.11